api/backup.go

```diff
@@ -95,7 +95,6 @@ func (s *Server) BackupDelete(w http.ResponseWriter, req *http.Request) error {
 	if err := s.m.DeleteBackup(input.Name, volName); err != nil {
 		return errors.Wrapf(err, "error deleting backup %v of volume %v", input.Name, volName)
 	}
-	logrus.Debugf("Removed backup %v of volume %v", input.Name, volName)
```

Comment:
The delete backup log in https://github.com/longhorn/longhorn-manager/blob/777d0b64b1c504dc178e7405b23d17af8e7cfd70/engineapi/backups.go#L244 already.